cloudrun-malware-scanner/server.js

```diff
@@ -43,29 +43,7 @@ const CLAMD_TIMEOUT = 600000;
 // Note scanning a 500MiB file can take 5 minutes, so ensure timeout is
 // large enough.
 const MAX_FILE_SIZE = 500000000; // 500MiB
```

Comment:
we are going to need increase this .. maybe make it into env var as well
some of the games are on the order of x gigs